web/tests/api-spec.js

```diff
@@ -53,5 +62,31 @@ describe('api service', function () {
         });
         
     });
+
+    describe('get', function () {
+        it('should return a promise with the data from the api', function () {
+            const url = 'http://localhost:8000/forms/1';
+
+            this.api.getForm(this.get.url).then((data) => {
+                expect(data).toEqual(this.get.OK.response_data);
+            });
+
+            this.$rootScope.$digest();
+            this.$httpBackend.flush();
+        });
+    });
+
+    describe('post', function () {
+        it('should return a promise with the data of a new response', function () {
+            const data = this.post.fields;
+
+            this.api.post(this.post.url, data).then((data) => {
```

Comment:
Maybe just use `this.post.fields` directly, as there's a confusing amount of `data`